lib/screens/login_screen.dart

```diff
@@ -92,63 +88,93 @@ class _LoginScreenState extends State<LoginScreen>
-          children: <Widget>[
-            ElevatedButton(
-              onPressed: () {
-                _controller.forward();
-                // Navigator.of(context).pushNamed('/mainscreen');
-              },
-              child: const Text('log in'),
-            ),
-            SlideTransition(
-              position: _offsetAnimation,
-              child: FadeTransition(
-                opacity: _controller,
-                child: Column(
-                  mainAxisSize: MainAxisSize.max,
-                  children: <Widget>[
-                    SizedBox(
-                      width: 200,
-                      child: TextField(
-                        controller: _loginController,
-                        decoration: const InputDecoration(
-                          border: OutlineInputBorder(),
-                          labelText: 'Login',
-                        ),
-                      ),
+        child: PicosBody(
+          padding: 0,
+          child: Padding(
+            padding: const EdgeInsets.all(10),
+            child: Column(
+              children: <Widget>[
+                const Image(
+                  image: AssetImage('assets/PICOS_Logo_RGB.png'),
+                ),
+                RichText(
+                  text: TextSpan(
+                    style: const TextStyle(
+                      height: 2,
+                      color: Colors.black,
                     ),
-                    const SizedBox(height: 10),
-                    SizedBox(
-                      width: 200,
-                      child: TextField(
-                        controller: _passwordController,
-                        obscureText: true,
-                        decoration: const InputDecoration(
-                          border: OutlineInputBorder(),
-                          labelText: 'Password',
+                    children: <TextSpan>[
+                      TextSpan(
+                        text:
+                            '${AppLocalizations.of(context)!
+                            .welcomeToPICOS},\n',
+                        style: const TextStyle(
+                          fontWeight: FontWeight.bold,
                         ),
                       ),
-                    ),
-                    ElevatedButton(
-                      onPressed: () => _submitHandler(
-                        _loginController.text,
-                        _passwordController.text,
-                        context,
+                      TextSpan(
+                        text:
+                            AppLocalizations.of(context)!
+                            .thankYouForParticipation,
                       ),
-                      child: const Text('Submit'),
+                    ],
+                  ),
+                ),
+                const SizedBox(
+                  height: 15,
+                ),
+                SizedBox(
+                  width: 200,
+                  child: TextField(
```

Comment:
I would suggest using PicosTextField, if you haven't tried it yet.
Same for the password field.